api/lorawan.proto

```diff
@@ -739,6 +739,10 @@ message MACCommand {
   }
 }
 
+message MACCommands {
```

Comment:
Maybe add a note that this is used (currently) only for internal purposes.
(If we use it in the future as part of incoming request, we'll probably need to add a limit).